tests/unittest/_torch/auto_deploy/unit/singlegpu/transformations/library/test_export_fp8_linear_to_onnx.py

```
@@ -0,0 +1,191 @@
+- Transpose + MatMul + Add (linear computation)
+"""
+
+import tempfile
+from pathlib import Path
+
+import onnx
+import pytest
+import torch
+
+# Import to register the custom op
+import tensorrt_llm._torch.auto_deploy.custom_ops.torch_quant  # noqa: F401
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)